docs/usage.rst

```diff
@@ -1265,6 +1265,119 @@ Besides the `invest` variable, new variables are introduced as well. These are:
       monthly periods, but you would need to be very careful in parameterizing your energy system and your model and also,
       this would mean monthly discounting (if applicable) as well as specifying your plants lifetimes in months.
 
+Modelling cellular energy systems
+---------------------------------
+
+The cellular approach is a concept proposed by the VDE-ETG. It is related to smart-grids and multi-microgrid systems but extends
+both. The idea is to group the components of an energy system into a hierarchically aggregating structure of cells. For example,
+the sources, sinks, storages and converters of a household could be a single cell. Then a group of physically neighboring 
+households could form another cell, made up of household-cells. This behaviour can be scaled up. The real game-changer in the 
+cellular approach is the way the cells are operated, which will not be covered here. Here, we focus on the way such cellular
+energy systems can be modeled.
+
+So far, the implementation in solph is just a neat way to group different parts of a larger energy system into cells. Further 
+functionality is expected to make its way into solph in the future, e.g. a distributed optimization procedure based on 
+Dantzig-Wolfe decomposition.
+
+Modelling procedure
+^^^^^^^^^^^^^^^^^^^
+
+Similar to the creation of regular energy systems, the creation of energy cells is the first step in model creation. Essentially,
+each energy cell is just an energy system, therefore we use the class :py:class:`oemof.solph.EnergySystem` to create energy cells.
+
+.. code-block:: python
+
+    es = EnergySystem(
+        label="es", timeindex=timeindex, infer_last_interval=False
+    )
+    ec_1 = EnergySystem(
+        label="ec_1", timeindex=timeindex, infer_last_interval=False
+    )
+    ec_2 = EnergySystem(
+        label="ec_2", timeindex=timeindex, inver_last_interval=False
+    )
+
+Now we can go on and add components to the energy cells just like we do with regular energy systems.
+
+.. code-block:: python
+
+    bus_el_es = buses.Bus(label="bus_el_es")
+    es.add(bus_el_es)
+
+    bus_el_ec_1 = buses.Bus(label="bus_el_ec_1")
+    sink_el_ec_1 = cmp.Sink(
```

Comment:
I think there is no standard import alias, is there? Thus I would replace `cmp` by `solph.components` and also place the import statement `import oemof.solph as solph` above to make it clear.